### PR TITLE
feat(Creepypasta): add presence

### LINKED ISSUE #5445: CreepyPasta | creepypasta.com
### Discussed in https://github.com/PreMiD/Presences/discussions/4479

<div type='discussions-op-text'>

<sup>Originally posted by **lonelyhikiko** July 25, 2020</sup>
**Prerequisites and essential questions** 
- [x] Is it a popular site? yes
- [x] Is the website older than 2 months?  i guess so , yeah
- [ ] Is the site locked to a specific country/region? i don't think so
- [ ] Is the site a paid service? (e.g. Netflix, Hulu) no
- [x] Does the website feature NSFW content? (e.g. porn, etc...) i don't know
- [ ] Are you a donator/patron?no
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority? yeah

**What's your Discord username?** GrapeJuice🍇#4469

**What's the name of the service?** https://www.creepypasta.com/

**What should the Presence display?** name of the website , the webpage and the story being read 

**If possible, please provide a logo for the service (512x512 minimum)** 
</div>

### PR DESCRIPTION
**Pull request by joerkig**
**Describe the changes in this pull request:**
add the creepypasta presence

resolves #5445

<details>
<summary> Proof to proving the creation/modification working </summary>
<br>

image
image
image
image
image
image
image
image
image
image
image
image

</details>

### CODE REVIEW

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
Can you not use genertic type casting instead?

**Comment by joerkig** [comment]
> Can you not use genertic type casting instead?

image

**Comment by joerkig** [comment]
> > Can you not use genertic type casting instead?
> 
> image

@EncryptedDev  would you prefer if I did querySelector with generic type casting?

File: websites/C/Creepypasta/presence.ts [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+    };
+
+  switch (document.location.pathname) {
+    case "/":
+      presenceData.details = "Viewing Home Page";
+      break;
+    case "/archive/top-ranked/":
+      presenceData.details = "Browsing Top-Ranked";
+      if (
+        (document.getElementsByName("_orderby")[0] as HTMLSelectElement)
```
Comment (Slowlife01) [inline websites/C/Creepypasta/presence.ts:23]: Can you make it a variable instead?

**Review by Slowlife01 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**